pkg/pillar/containerd/oci_test.go

```diff
@@ -6,19 +6,20 @@ package containerd
 import (
 	"encoding/json"
 	"fmt"
-	zconfig "github.com/lf-edge/eve/api/go/config"
```

Comment:
This feels like a spurious change -- can we drop it?

Reply:
Haha! I didn't even notice it. It's automatic gofmt formatting.

Reply:
So, of course, I wrote `gofmt`, when that is wrong and was unintended. It is `goimports`, which most IDEs use, at least as an option (including VS Code and Atom). 

In addition to ensuring that everything necessary is imported, and removing unnecessary imports, `goimports` adds at least one important layer to `gofmt`.

* `gofmt` sorts your imports alphabetically _within a group_, but has no opinion about grouping itself.
* `goimports`, in addition to sorting within groups, separates your imports into 2 groups: stdlib, and external. Within each group, it sorts within groups.

What we are seeing in this PR is the automatic application of goimports rules to the file (in this case, by an IDE, but could be manual too).

The resultant output remains gofmt-compliant, but also complies with goimports rules. Examples from [kubernetes](https://github.com/kubernetes/kubernetes/blob/master/pkg/controlplane/controller.go#L19-L43) and [containerd](https://github.com/containerd/containerd/blob/master/content/content.go#L19-L26).

I think we should leave it as is; improved readability at no cost to us.